[README] Add password expiration

Ana06 opened the pull request:
A password expiration blocks the FLARE-VM installation as it asks to change the password after a restart. I had this issue today. Document how to set `Password never expires` in the user properties in `lusrmgr.msc` to avoid the issue.

emtuls commented:
Is this something that we could automate and do early during the installation process? 

It seems to just require running `WMIC USERACCOUNT WHERE Name='<username here>' SET PasswordExpires=FALSE`, but I could be wrong.

Ana06 commented:
@emtuls 
> It seems to just require running WMIC USERACCOUNT WHERE Name='<username here>' SET PasswordExpires=FALSE, but I could be wrong.

have you tried this? It doesn't work for me:
![image](https://github.com/mandiant/flare-vm/assets/16052290/558fcf52-01ab-439b-a226-e86ce52b2357)

emtuls commented:
Whoops. That is how you would do it in `Command Prompt`. 
It would be a little different if doing it from Powershell: `Set-LocalUser -Name "<username here>" -PasswordNeverExpires $true `

**Command Prompt**
![image](https://github.com/mandiant/flare-vm/assets/35854880/3c4a262c-195e-46c6-9d77-07122c6d6068)

**Powershell**
![image](https://github.com/mandiant/flare-vm/assets/35854880/c23aeac9-1c9c-4331-8840-14584c5d7384)

Ana06 committed:
[installer] Simplify getting user name

Use `$Env:UserName` to get the user name. Environment variables can be
modified by the user, but for our check, I think it is good enough and
simpler than the previous approach.

Ana06 committed:
[installer] Set password to never expire

A password expiration blocks the FLARE-VM installation as it asks to
change the password after a restart.

emtuls reviewed (approved):
Looks good! Thanks for making this happen!
